app/pages/supplier/dashboard/template.njk

```diff
@@ -2,39 +2,82 @@
 {% from 'components/button/macro.njk' import button %}
 {% from '../../common/dashboard/components/dashboard-sectionGroup.njk' import dashboardSectionGroup %}
 {% from '../../../common/components/error-summary/macro.njk' import errorSummary %}
+{% from 'components/tag/macro.njk' import tag %}
+{% from 'components/view-data-bulletlist/macro.njk' import viewDataBulletlist %}
 
 {% block body %}
-<div data-test-id="dashboard" class="nhsuk-grid-row">
-  <div class="nhsuk-grid-column-full">
+<div data-test-id="dashboard">
+  <div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-full">
+      {% if (errors.length > 0) %}
+        {{ errorSummary({
+          errors: errors
+        }) }}
+      {% endif %}
+    </div>
+    <div class="nhsuk-grid-column-two-thirds">
 
-    {% if (errors.length > 0) %}
-      {{ errorSummary({
-        errors: errors
-      }) }}
-    {% endif %}
-
-    <h1>Marketing Page - Dashboard</h1>
-    <div data-test-id="dashboard-preview-button">
-      {{ button({
-        "text": "Preview Marketing page",
-        "classes": "nhsuk-button--secondary",
-        "href": previewUrl
+      {{ tag({
+        text: 'Supplier name',
+        classes: 'nhsuk-u-margin-bottom-4 bc-c-tag-outline'
       }) }}
+
+      <h1>&lt;Catalogue Solution name&gt; listing page – all sections</h1>
```

Comment:
do you want the < and > and hardcoded solution name? looks a bit weird

Reply:
Was just to match the designs, I can't pull through the solution name yet so I thought it looked more obvious it needed to be sorted with the <>

Reply:
the solution name comes back from the API ```{
  id: '100000-001',
  name: 'Write on Time',
  sections: {...}
}```
so you can get it from there? Just need to extract it from the data that comes in in the controller file